src/plugins/workspace/public/components/workspace_updater/workspace_updater.tsx

```diff
@@ -139,6 +180,7 @@ export const WorkspaceUpdater = () => {
           restrictWidth
           pageTitle="Update Workspace"
           rightSideItems={[
+            <EuiButton onClick={() => exitWorkspace()}>Exit</EuiButton>,
```

Comment:
```suggestion
            <EuiButton onClick={exitWorkspace}>Exit</EuiButton>,
```
exitWorkspace, do not create a new arrow function here.